include/swift/AST/DiagnosticsParse.def

```diff
@@ -692,12 +689,15 @@ ERROR(multiple_parameter_ellipsis,none,
       "only a single variadic parameter '...' is permitted", ())
 ERROR(parameter_vararg_default,none,
       "variadic parameter cannot have a default value", ())
-ERROR(parameter_inout_var_let,none,
+ERROR(inout_as_attr_disallowed,none,
+      "'inout' before a parameter name is not allowed, place it before the parameter type instead",
+      ())
+ERROR(parameter_inout_var_let_repeated,none,
       "parameter may not have multiple 'inout', 'var', or 'let' specifiers",
       ())
+ERROR(parameter_let_as_attr,none,
+      "'let' as a parameter attribute is not allowed", (bool))
 
```

Comment:
The (bool) parameter is not used and is always false.  You can drop it.  I was thinking that you can do something like:

"'%select{let|var}0' as a parameter attribute is not allowed"

Where the bool is "isVar()"